run.go

```diff
@@ -193,6 +197,9 @@ func (d *PlaywrightDriver) DownloadDriver() error {
 
 func (d *PlaywrightDriver) run() (*connection, error) {
 	cmd := exec.Command(d.DriverBinaryLocation, "run-driver")
+	if d.options != nil {
```

Comment:
I feel like this should be a default. So the user does not have to pass it over. We do it in the ports too:
- Python: https://github.com/microsoft/playwright-python/blob/59369fe126f49c10597d5c9099840bdc8ccdcf15/playwright/_impl/_transport.py#L121
- .NET: https://github.com/microsoft/playwright-dotnet/blob/eade3f5894752dc64cc1c242a37f03b39355c625/src/Playwright/Transport/StdIOTransport.cs#L122
- .NET: https://github.com/microsoft/playwright-dotnet/blob/eade3f5894752dc64cc1c242a37f03b39355c625/src/Playwright/Program.cs#L55-L57

Reply:
There is no such processing now. Will you consider adding it by default in the future?

Reply:
I added a defaultSysProcAttr, what do you think?